### PR TITLE
2369 update faraday gem

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
After an uptick of faraday constraints within _breakers_ and _betamocks_, this PR upgrades Faraday within vets-api to version 0.17.0 (previous version 0.9.2) 

## Testing done
Specs passing

Tests updated:
  -Randomly generated multi-part boundaries solution was addressed in relation to unique multi-part boundary string mismatching for VCR request cassettes

#### Unique to this PR
- [x] Tests passing

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,7 +32,7 @@ gem 'attr_encrypted', '3.1.0'
 gem 'aws-sdk-s3', '~> 1'
 gem 'aws-sdk-sns', '~> 1'
 gem 'betamocks', git: 'https://github.com/department-of-veterans-affairs/betamocks', branch: 'master'
-gem 'breakers'
+gem 'breakers', git: 'https://github.com/department-of-veterans-affairs/breakers', branch: 'master'
```
Comment (omgitsbillryan) [inline Gemfile]: Are we not allowed to run a new release of the breakers gem?

File: spec/support/vcr_multipart_matcher_helper.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#This is needed for mismatching multi-part boundaries -- replacing random boundaries
+#Multipart boundaries are most often randomly generated by the HTTP library or browser
+#so when VCR tries to match a multipart request with a cassette, the boundaries will be different and the match will fail.
+#This was introduced when upgrading to Faraday 0.17.0
+
+#ex -----------boundary=RubyMultipartPost-some_random_boundary_string
```
Comment (omgitsbillryan) [inline spec/support/vcr_multipart_matcher_helper.rb]: Love the comment, but could you expand on this example a bit - maybe provide a before and after of what it looks like once `normalized_multipart_request` runs?

File: Gemfile (hunk 1 above)
Comment (LindseySaari) [reply Gemfile]: When running bundle update on breakers, I receive "Bundler attempted to update breakers but its version stayed the same" unless the repo/master branch is specified. Any ideas?

File: spec/support/vcr_multipart_matcher_helper.rb (hunk 2 above)
Comment (LindseySaari) [reply spec/support/vcr_multipart_matcher_helper.rb]: Done ✔️

File: Gemfile (hunk 1 above)
Comment (kfrz) [reply Gemfile]: I don't have maintainer permissions to build/push the new gem version but I emailed Aubrey Holland. 

https://rubygems.org/gems/breakers

Comment (saneshark) [reply Gemfile]: Aubrey was former Ad Hoc employee. I'm thinking this gem could probably be forked and made available in DSVA github.

**Comment by kfrz** [comment]
Sounds like a plan - would it be cleaner if we could get it transferred
directly from Aubrey?

*Keifer Furzland*
Software Engineer
218-349-7107
*Oddball.io <http://oddball.io/> *| A strangely human digital agency

**Pull request merged**